lib/health_cards/issuer.rb

```diff
@@ -1,33 +1,50 @@
 # frozen_string_literal: true
 
-require_relative 'file_key_store'
-
-require 'json/jwk'
-
 module HealthCards
-  # Verifiable Credential Issuer
-  #
-  # https://www.w3.org/TR/vc-data-model/#issuer
+  # Issue Health Cards based on a stored private key
   class Issuer
-    def initialize(key_store)
-      @key = key_store.load_or_create_key
+    attr_reader :url, :key
+
+    # Create an Issuer
+    #
+    # @param key [HealthCards::PrivateKey] the private key used for signing issued health cards
+    def initialize(url: nil, key: nil)
```

Comment:
I don't think `key` should have a default value since an error is raised if it's `nil`.

Reply:
I think I got them all unless there's more hidden comments...